adam-r/bdgenomics.adam/DESCRIPTION

```diff
@@ -1,4 +1,4 @@
-Package: bdg.adam
+Package: bdgenomics.adam
 Type: Package
 Version: 0.23.0
 Title: R Frontend for Big Data Genomics/ADAM
```

Comment:
Sorry, missed this before, the Description line below needs updating, it refers to SparkR.